src/server/views/layouts/partials/head.njk

```diff
@@ -4,15 +4,15 @@
 <meta name="name" content="DVSA RSP" />
 <meta name="theme-color" content="#0b0c0c" />
 <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
-<meta property="og:image" content="{{ assets }}/images/opengraph-image.png?0.22.1">
+<meta property="og:image" content="{{ assets }}/images/govuk-opengraph-image.png?0.22.1">
```

Comment:
I'd imagine the purpose of that query string is to bust the cache. Recommend we bump it as part of this PR too.

Reply:
done